bin/add-license-headers.sh

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env bash
+
+TARGET_DIR=""
+FIND_COMMAND_OPTS=""
+SUPPORTED_FILE_TYPE="*" # Default is to choose all file types
+
+EXIT_CODE=0
+
+# Global flags
+CHECK_ONLY_FLAG=0
+HELP_FLAG=0
+VERBOSE_FLAG=0
+
+LICENSE_TEXT="/*
+ * Copyright 2019 Comcast Cable Communications Management, LLC
```

Comment:
this copyright doesn't align with the others.  Either choose 2018 or 2019, but stick with one